docs/new/common/packages.md

```diff
@@ -27,7 +27,8 @@ substreams gui \
  https://spkg.io/streamingfast/erc20-balance-changes-v1.1.0.spkg \
  map_balance_changes \
  -e mainnet.eth.streamingfast.io:443 \
- --start-block 1397553
+ --start-block 1397553 \
+ --network eth
```

Comment:
Similar changes for here.

Reply:
b035cd45938ddd264c9664ad4300ca0be40dfd73